### PR TITLE
Made top banner + tests

### PR DESCRIPTION
**Pull request by FlorisSp**
screenshot from 2017-03-28 16-09-36

### CODE REVIEW

File: spec/views/public.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe "pages/public.html.erb" do
+  it "contains \"Taste of Code\"" do
+    render
+      response.should have_text("Taste of Code")
```
Comment (miriamtocino) [inline spec/views/public.html.erb_spec.rb]: You don't need to indent this one extra tab. Please align with `render` line 😄

File: spec/views/public.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  end
+
+  it "contains \"Beginner coding workshop\"" do
+    render
+      response.should have_text("Beginner coding workshop")
```
Comment (miriamtocino) [inline spec/views/public.html.erb_spec.rb]: See my previous comment ^^

File: app/views/pages/public.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<section class="jumbotron text-center">
+  <div class="container">
+    <h1 class="jumbotron-heading">Taste of Code</h1>
+    <p class="lead text-muted">Beginner coding workshop</p>
+  </div>
+</section>
```
Comment (miriamtocino) [inline app/views/pages/public.html.erb]: I still don't see how you are rendering this page.
I just checked your routes and there is no `/public` URL in there.

Will you be adding that in another pull?

Comment (FlorisSp) [inline app/views/pages/public.html.erb]: We're waiting for the models and will update the routes after that!

**Comment by miriamtocino** [comment]
Looks good to me!!

:shipit: 

^^ this emoji means `ship it == merge it` !!

**Pull request merged**